Fedoraware/TeamFortress2/TeamFortress2/Features/PlayerResource/PlayerResource.cpp

```diff
@@ -18,40 +18,34 @@ void C_PlayerResource::Update() {
 	}
 }
 
-int C_PlayerResource::GetHealth(CBaseEntity* player) {
+int C_PlayerResource::GetHealth(int idx) {
```

Comment:
> Still not sure why we do player resource in this way...

Because we don't need the player's entity pointer, we only need the index to get the value.